pymatgen/analysis/phase_diagram.py

```diff
@@ -599,14 +599,98 @@ def get_equilibrium_reaction_energy(self, entry):
             equilibrium reaction energy <= 0.
         """
         if entry not in self.stable_entries:
+            # NOTE Comprhys: could rasing a warning and returning NaN be
+            # a better solution here than raising an Error? an Error interupts a
+            # whole program even if it could complete with a NaN.
             raise ValueError("Equilibrium reaction energy is available only "
                              "for stable entries.")
         if entry.is_element:
             return 0
         entries = [e for e in self.stable_entries if e != entry]
         modpd = PhaseDiagram(entries, self.elements)
-        return modpd.get_decomp_and_e_above_hull(entry,
-                                                 allow_negative=True)[1]
+        return modpd.get_decomp_and_e_above_hull(entry, allow_negative=True)[1]
+
+    def get_decomposition_energy(self, entry):
+        """
+        NOTE a different name might be better to avoid confusion with
```

Comment:
I'm ok with a different name if this is kept, don't have one to recommend though (either way, remove the NOTE from the docstring before merging)

Reply:
Though if renaming just make sure to add a note to the paper citation below in `_slsqp_decomp_solution`, along the lines of "this is referred to as the decomposition enthalpy in [...]"

Reply:
I was thinking maybe `get decomp and pseudo e to hull' as an alternative name